src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -58,13 +58,17 @@ public void ConfigureServices(IServiceCollection services)
             services.AddIdentity<ApplicationUser, IdentityRole>()
                 .AddEntityFrameworkStores<ApplicationDbContext>();
 
+            var urlPrefix = _configuration.GetValue<string>("urlPrefix");
+
             services.AddIdentityServer(options =>
                 {
                     options.Events.RaiseFailureEvents = true;
                     options.Events.RaiseErrorEvents = true;
                     options.Events.RaiseSuccessEvents = true;
                     options.IssuerUri = issuerUrl;
-                    options.UserInteraction.ErrorUrl = "/Error";
+                    options.UserInteraction.LoginUrl = $"/{urlPrefix}/account/login";
```

Comment:
Consistent casing? Just comparing Account to Error. 

Also if someone configures URL Prefix to an empty string or NULL to remove the prefix, would the application still work? I am not sure it would cope with `//account/login`. Look up PathString it may help in this scenario to chain the segments of a path.